### DISCUSSION TITLE
Be able to use edge as a webbrowser

### DISCUSSION THREAD

**Discussion by Mrgaton** [dotnet/winforms#11389] [Ideas] [answered]
Imagine instead of using the crapi webbrowser use edge webview with a lot of features would be awesome

**Comment by Mrgaton** [comment]
![image](https://github.com/dotnet/winforms/assets/68958481/35d6491a-6bfd-4968-bda0-df6c1648b01d)

Cause this is horrible

**Reply by RussKie** [reply]
Related: https://github.com/dotnet/winforms/issues/6964

**Comment by Mrgaton** [comment]
And then be able to call functions from the code from web :0 or hook from the code functions from the web

**Comment by paul1956** [comment, marked as answer]
WebWiew2 works great with WinForms but if you need read/write access to the HTML it is much more difficult then WebBrowser.  I have no idea why it’s so difficult if that could be fixed I would switch everything to WebView2.

**Reply by Mrgaton** [reply]
The problem is that I am using framework 4.8.1

**Reply by RussKie** [reply]
WebView2 is supported on [.NET Framework 4.6.2+](https://learn.microsoft.com/en-au/microsoft-edge/webview2/?form=MA13LH#supported-platforms)
